controllers/topics.controller.js

```diff
@@ -0,0 +1,12 @@
+const { readAllTopics } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+  console.log("hello from the controller");
```

Comment:
remember to remove console logs and un-necessary comments from your code when submitting a pull request :)